### ISSUE TITLE
Issue with Animal Crossing New Leaf Welcome Amiibo

### ISSUE THREAD

**Issue by benduckwdesign** [meladroit/svdt#36]
Backing up the save data works great. However, upon trying to restore, it throws an error:

! Error rewriting secure value.
[result code ffffffff]

Upon opening New Leaf, the game does not accept the save data, so obviously it didn't rewrite the secure value correctly (as it states.)

Any help is appreciated.

**Comment by nofaithhere** [comment]
Same issue!

**Comment by nofaithhere** [comment]
@miniluigi008 I found a work around if you need help

**Comment by benduckwdesign** [comment]
@nofaithhere My only solution right now is to use the most recent save backup, but I have a cartridge and eShop version that I play, so I tend to replace the save of whichever one I have been playing most recently (cartridge to eShop or eShop to cartridge.) Without secure value rewriting I can't do this, and I can't restore from really old saves either (they would have to be ACNLWA saves, of course.)

I'm open to any work arounds.

**Comment by nofaithhere** [comment]
@miniluigi008 svdt is definitely not working for secure value write on ACNLWA. What I ended up doing was creating a new save file in game. I used Secure Value Fixer to write the secure value from the new save to old save, then used AC:NL save manager to inject it back into my copy of the game.

It at least allowed me to recover my town, but won't help with the way you're going back and forth with saves (it would be a pain to have to start up a new town, finish the first objectives and then save the game just so you can write the secure value to a different save)

**Comment by suloku** [comment]
ACNLWA is a new game, and the adress for it's secure value isn't in the secure value database (asr.dat).

JKSaveManager now has a 3dsx version. It auto-deletes secure value from userland, so using JKSaveManager should work. That feature is something that should be added to svdt, but I don't have the time right now.

Can you upload a welcome amiibo savegame? Since it's an update, maybe they changed something regarding the savegame, even if secure value failed the error you get is strange.

EDIT: I noticed using the pc program works, so I wonder why this bug occurs...

**Comment by benduckwdesign** [comment]
@suloku Sorry for the year wait, I ended up breaking the lower screen ZIF connector off of my main 3DS motherboard. A lack of a lower screen is not only nonfunctional, but unfortunately it does not allow the console to boot. My fixed solution involves lots of stacked double sided sticky tape which creates enough pressure for the ribbon cable to come into contact with the board.

I will upload a welcome amiibo savegame by tonight.

I also recently upgraded to boot9strap, so I will be trying out JKSaveManager also.
